### ISSUE TITLE
Archive parameter description missing

### ISSUE THREAD

**Issue by SetTrend** [MicrosoftDocs/office-docs-powershell#7683]
This page - and related pages - lack a description of the `Archive` parameter:

![No description](https://user-images.githubusercontent.com/9283914/115962569-547bd100-a51c-11eb-8ead-2a392e4390a3.png)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 573e63b4-b27b-5b7b-9269-78d308fd64d3
* Version Independent ID: 57b35909-0381-1181-58bb-2262c33576d1
* Content: [Set-MailboxRegionalConfiguration (ExchangePowerShell)](https://docs.microsoft.com/en-us/powershell/module/exchange/set-mailboxregionalconfiguration?view=exchange-ps)
* Content Source: [exchange/exchange-ps/exchange/Set-MailboxRegionalConfiguration.md](https://github.com/MicrosoftDocs/office-docs-powershell/blob/master/exchange/exchange-ps/exchange/Set-MailboxRegionalConfiguration.md)
* Service: **exchange-powershell**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by chrisda** [comment]
@SetTrend, see the discussion in <https://github.com/MicrosoftDocs/office-docs-powershell/issues/7429> about empty parameter descriptions. Do you use this parameter? Does it work as you expect?

**Comment by SetTrend** [comment]
Good question ...

The issue is: I don't know. If there'd be a description, then I could tell whether it's important or not, yet without, I'm lost.

At this stage I can't tell wether omitting it will demolish my Active Directory settings due to incorrect parameters set. This is particularly true for Exchange Online.

Once settings are demolished, this could seriously jeopardize the company. And that would cause fear. And, as we all know, fear leads to anger, anger leads to hate, hate leads to suffering.

Basically, that's the core if this issue.

**Comment by chrisda** [comment]
@SetTrend, in looking at Get-MailboxRegionalConfiguration and Set-MailboxRegionalConfiguration, Archive is a switch parameter (no values). 

Judging by how the Archive switch is described in the 21 other cmdlets that have it (listed below), it appears to be a simple identifier, as in: don't return/modify the specified mailbox; return/modify the archive mailbox that's associated with the mailbox instead. If you don't have archive mailboxes, it would be pretty useless.

Availability vs. functionality is the main question: it appears to be publicly available, so I added it to the topic (feel free to confirm or deny). Does it actually do anything? You can test with Get-MailboxRegionalConfiguration. My guess would be that when an archive mailbox is created, it uses the regional settings of the user's mailbox, so the only useful scenario would be when the user's mailbox and their archive mailbox use different date-time and/or language settings.

- Connect-Mailbox
- Disable-Mailbox
- Disable-RemoteMailbox
- Enable-Mailbox
- Enable-RemoteMailbox
- Export-MailboxDiagnosticLogs
- Get-EXOMailbox
- Get-EXOMailboxFolderStatistics
- Get-EXOMailboxStatistics
- Get-FailedContentIndexDocuments
- Get-Mailbox
- Get-MailboxFolderStatistics
- Get-MailboxRepairRequest
- Get-MailboxStatistics
- Get-RemoteMailbox
- New-Mailbox
- New-MailboxRepairRequest
- New-RemoteMailbox
- Test-ExchangeSearch
- Test-MAPIConnectivity
- Test-OutlookConnectivity

**Comment by SetTrend** [comment]
Thank you for taking the time, @chrisda, and dealing with the matter.

OK, archive mailboxes may become a necessity to consider later then, when someone in our company starts to be using it.

I wonder: don't they use a linter in the ExchangePowerShell module build pipeline that's validating parameters for missing descriptions? ... My particular case reported here may be harmless, but I'm pretty sure that if data is getting lost and outages occur, someone might want to take MS liable for data loss due to lack of (or false) parameter documentation.

**Comment by chrisda** [comment]
@SetTrend, as I proposed in the link earlier in the issue, is it better to have an available parameter present but undocumented in a topic, or to have an available parameter not present in the topic at all? Pros and cons either way; I'm not sure there's a definitive answer.

**Comment by SetTrend** [comment]
> I'm not sure there's a definitive answer.

Yes, I agree. Though, I would claim that an undocumented parameter is just like an non-existing parameter, as it cannot be applied wisely and deliberately (except by the single one person who implemented it).
